galleries/tutorials/histogram_normalization.py

```diff
@@ -1,27 +1,28 @@
 """
 .. redirect-from:: /gallery/statistics/histogram_features
 
-===================================
-Histogram bins, density, and weight
-===================================
+.. _histogram_normalization:
+
+==================================
+bins, density, and weights in hist
+==================================
 
 The `.Axes.hist` method can flexibly create histograms in a few different ways,
 which is flexible and helpful, but can also lead to confusion.  In particular,
-you can:
+the method has the following parameters:
 
-- bin the data as you want, either with an automatically chosen number of
+- ``bins`` bin the data as you want, either with an automatically chosen number of
   bins, or with fixed bin edges,
-- normalize the histogram so that its integral is one,
-- and assign weights to the data points, so that each data point affects the
-  count in its bin differently.
+- ``density`` normalize the histogram so that its integral is one,
+- ``weights`` assign weights to each data point so that each point affects the
+  count differently.
```

Comment:
Now that you have a parameter name here, the verbs don't match. Or maybe this would do better as a definition list?